Sources/Core/Shared/Condition.swift

```diff
@@ -166,7 +166,7 @@ public class ComposedCondition<C: Condition>: Condition, AutomaticInjectionOpera
         super.init()
         mutuallyExclusive = condition.mutuallyExclusive
         name = condition.name
-        injectResultFromDependency(condition) { operation, dependency, _ in
+        let _ = injectResultFromDependency(condition) { operation, dependency, _ in
```

Comment:
Oh, is this now a warning in b2? Would probably be better to add the appropriate attribute to the method.